packages/client/foreman_scap_client_bash/foreman_scap_client_bash.spec

```
@@ -0,0 +1,50 @@
+%global repo_orgname theforeman
+%global repo_name foreman_scap_client_bash
+%global client_name foreman_scap_client
+%global config_dir %{_sysconfdir}/%{client_name}
+
+Name: foreman_scap_client_bash
+Version: 0.2.0
+Release: 1%{?dist}
+Summary: Client script that runs OpenSCAP scan and uploads the result to foreman proxy
+
+License: GPLv3
+URL: https://github.com/theforeman/foreman_scap_client_bash
+Source0: https://github.com/%{repo_orgname}/%{repo_name}/releases/download/v%{version}/%{repo_name}-%{version}.tar.gz
+
+Requires: bash
+Requires: curl
+%if 0%{?suse_version}
+Requires: openscap-utils
+%else
+Requires: openscap-scanner
+%endif
+Requires: bzip2
+BuildArch: noarch
+
+Obsoletes: rubygem-foreman_scap_client
```

Can we please have an versioned obsolete, so we can come back to Ruby if things go south?
```suggestion
Obsoletes: rubygem-foreman_scap_client < 0.6.2-2
```

---

https://github.com/theforeman/foreman-packaging/pull/11636